### PR TITLE
backend challange from pt-br to eng

### PR DESCRIPTION
**Pull request by femariotti**

### CODE REVIEW

**Comment by femariotti** [comment]
@MarcoWorms  @felipeblassioli

File: challenges/pt-br/back-challenge.md [hunk 1]
```diff
@@ -48,20 +52,31 @@ Usuário exemplo:
...
+3. You demonstrate the knowledge on how to test the critical parts of the application. We **do not require** 100% coverage.
+4. The application is well and logically organised.
+5. The submission is accompanied by documentation with the reasoning on the decisions taken.
+6. The code is documented and is easy to follow.
+7. Following the industry standard style guide.
+8. A git history (even if brief) with clear, concise commit messages.
+​
 
-* Caso o serviço 1 retorne um erro, a lista de produtos ainda precisa ser retornada, porém com esse produto que deu erro sem desconto.
+Before you start, take a look at our [key values](https://www.keyvalues.com/hash). It can show you how we think and work in terms of development. Also do your best! We'll evaluate it like you did.
```
Comment (felipeblassioli) [inline challenges/pt-br/back-challenge.md]:
```suggestion
Also if you'd like to know more about our developer culture, how we think and work in terms of development. check out our  [key values](https://www.keyvalues.com/hash).
```

**Review by felipeblassioli — CHANGES_REQUESTED** [review]

File: challenges/pt-br/back-challenge.md (hunk 1 above)
Comment (femariotti) [reply challenges/pt-br/back-challenge.md]: > 

Also if you'd like to _know_ more about our developer culture, how we think and work in terms of development. check out our  [key values](https://www.keyvalues.com/hash).

Comment (felipeblassioli) [reply challenges/pt-br/back-challenge.md]: Fixed!

**Pull request closed without merging**